### ISSUE TITLE
write unhandled-cases-workaround as mentioned in #24

### ISSUE THREAD

**Issue by lspitzner** [lspitzner/brittany#28]

**Comment by tfausak** [comment]
Do this mean _automatically_ passing unknown constructs through? Because the particular case mentioned in #24 (dealing with arrows) can be worked around using `--brittany-disable-next-binding`. For example:

``` hs
{-# language Arrows #-}
-- brittany-disable-next-binding
a = proc b -> do
  c <- d -< e
  pure -< f
```

**Comment by lspitzner** [comment]
> Do this mean automatically passing unknown constructs through?

Yes, and this was implemented in d0256bb0dba8d085ef2c19a48b46b79d5159abb0.
